src/MessageBird/Common/HttpClient.php

```diff
@@ -219,7 +219,7 @@ public function performHttpRequest($method, $resourceName, $query = null, $body
 
         // Split the header and body
         $parts = explode("\r\n\r\n", $response, 3);
-        list($responseHeader, $responseBody) = ($parts[0] === 'HTTP/1.1 100 Continue') ? array ($parts[1], $parts[2]) : array ($parts[0], $parts[1]);
+        list($responseHeader, $responseBody) = (strpos($parts[0], "\n") == false && strpos($parts[0], 'HTTP/1.') == 0) ? array ($parts[1], $parts[2]) : array ($parts[0], $parts[1]);
```

Comment:
Could you extract the condition into a separate variable like `$isThreePartResponse` for more readability?

Reply:
Also, please use strict type comparison.

Reply:
Yup, agreed! I have updated the code.